src/core/containers/HandleLogin/index.js

```diff
@@ -33,7 +34,8 @@ class HandleLogin extends React.Component {
   }
 
   render() {
-    const { code, state } = this.props.location.query;
+    const { location } = this.props;
+    const { code, state } = location.query;
     const { error } = this.state;
```

Comment:
well, I guess it's a boolean but I don't see any message passed to `LoginPage` about the error.